gr-utils/modtool/templates/gr-newmod/.gitignore

```
@@ -0,0 +1,5 @@
+*~
+*.pyc
+*.pyo
+build*/
+example/*.py
```

```suggestion
```
This line assumes that there will be no Python-based examples. Maybe we can have an entry like
```suggestion
examples/grc/*.py
```
instead?

Also, it's example*s*.

---

Yes - that makes sense!